### PR TITLE
Updating test matrix

### PR DESCRIPTION
**Pull request by nicrowe00**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -49,19 +56,23 @@ jobs:
         run: |
           set -euo pipefail
           cat /etc/os-release
-          if grep fedora /etc/os-release ; then
-            if [[ ${{ matrix.dotnet_version }} == 9.* ]]; then
-              dnf install 'dnf-command(copr)' -y
+          if [[ ${{ matrix.dotnet_version }} == 9.* ]]; then
```
Comment (tmds) [inline .github/workflows/ci.yml:59]: When reviewing this there are some things the reviewer needs to know:
- .NET 9 is a preview, that's why it needs copr
- .NET 9 introduced the sdk-aot package
- /etc/os-release is going to match the grep fedora also on Fedora derived OSes (like CentOS)
- .NET 8 introduced the dbg packages

Perhaps we can include this information with some comments.

**Comment by tmds** [comment]
@nicrowe00 feel free to merge if/when the CI is happy with the changes. Thanks!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (nicrowe00) [reply .github/workflows/ci.yml:59]: Thanks for the suggestion, I'll add it now.

**Review by omajid — APPROVED** [review]
Thanks!

**Pull request merged**